src/components/QueryResult.vue

```diff
@@ -70,12 +79,38 @@ export default Vue.extend( {
 		color: $font-color-subtle;
 		font-weight: $font-weight-style-description;
 		line-height: $font-line-height-style-description;
+		width: $dimension-width-full;
+		border: $border-width-thin $border-style-base $border-color-base-subtle;
+		border-radius: 0 0 $border-radius-base $border-radius-base;
 		text-align: center;
 	}
 
+	.querybuilder__result__header {
+		display: flex;
+		align-items: center;
+		background: $background-color-neutral-default;
+		border: $border-width-thin $border-style-base $border-color-base-subtle;
+		border-radius: $border-radius-base $border-radius-base 0 0;
+		box-sizing: border-box;
+		padding-inline-start: $dimension-spacing-xlarge;
+		// TODO: Remove this once we have the share button and use padding instead
+		height: 3em;
+		border-bottom: none;
+
+		& h2 {
+			font-family: $font-family-style-heading-sans;
+			font-style: normal;
+			font-weight: $font-weight-style-h5;
+			font-size: $font-size-style-h5;
+			color: $font-color-base;
+		}
+	}
+
 	.querybuilder__result__iframe {
 		width: $dimension-width-full;
-		border: none;
+		background: $background-color-neutral-default;
```

Comment:
```suggestion
		background: $color-base-90;
```

Reply:
👆🏻 same as above

Reply:
Oh this one doesn't even need it. We can't set background of the iframe (cleaning up the result css is another thing altogether which I actually love to do)

Reply:
yep, I wondered about it!